src/shim.js

```diff
@@ -2,6 +2,36 @@ import webpage from "webpage";
 import system from "system";
 
 /**
+ * Ensure that Function has bind() method (PhantomJS version <= 1.9 support)
+ * This is a Polyfill replacement from MDN.
+ * @see {@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/bind#Polyfill}
+ */
+/* eslint-disable no-extend-native, consistent-this, require-jsdoc, no-empty-function, no-invalid-this */
```

Comment:
I'm afraid so too.

I tried to modify the code to comply with the existing rules but this seems to be the minimum set of rules to disable.
However, they're re-enabled just after the added method so they cause no harm to the rest of code.